libs/core/langchain_core/tracers/event_stream.py

```diff
@@ -122,7 +122,7 @@ async def tap_output_aiter(
         # atomic check and set
         tap = self.is_tapped.setdefault(run_id, sentinel)
         # wait for first chunk
-        first = await py_anext(output, sentinel)
+        first = await py_anext(output, default=sentinel)
```

Comment:
hmm why?

Reply:
Convention that i always follow with python, if defined as named argument, then invoke as a named argument, if defined as positional argument then invoke as a positional argument. This is from before `*` days.

```
def py_anext(
    iterator: AsyncIterator[T], default: Union[T, Any] = _no_default
) -> Awaitable[Union[T, None, Any]]:
```